beamerthemeuulm.sty

```diff
@@ -628,10 +628,10 @@
     },
     % style definition for the normal boxes
     % #1: extra arguments, #2: title, #3: color of the frame and background, #4: boxname
-    uulm@boxstyle/.style n args=4{uulm@basic-boxstyle={#1}{#2}{#3}{1mm}{#4}, colback=#3!10},
+    uulm@boxstyle/.style n args=4{uulm@basic-boxstyle={#1}{#2}{#3}{1mm}{#4}, colback=#3!10!uulm@background},
     % style definition for their tight variant
     % #1: extra arguments, #2: title, #3: color of the frame, #4: boxname
-    uulm@tightboxstyle/.style n args=4{uulm@basic-boxstyle={#1}{#2}{#3}{0mm}{#4}, colback=white}
+    uulm@tightboxstyle/.style n args=4{uulm@basic-boxstyle={#1}{#2}{#3}{0mm}{#4}, colback=uulm@background}
```

Comment:
If you dislike the style of tightboxes, we can change that easily:

```suggestion
    uulm@tightboxstyle/.style n args=4{uulm@basic-boxstyle={#1}{#2}{#3}{0mm}{#4}, colback=white}
```

To get:

![Screenshot_20230615_115949](https://github.com/SoftVarE-Group/SlideTemplate/assets/9303573/66aff21d-a63f-4529-a48e-9bea42b89d37)


Or something like:

```suggestion
    uulm@tightboxstyle/.style n args=4{uulm@basic-boxstyle={#1}{#2}{#3}{0mm}{#4}, colback=uulm@boxmix}
```

To get a different shade:
![Screenshot_20230615_120113](https://github.com/SoftVarE-Group/SlideTemplate/assets/9303573/0abe3b9f-4fe3-4a5c-a07e-aee10dbdd148)